core/data-resolver/CategoryService.ts

```diff
@@ -16,6 +16,7 @@ const getCategories = async ({
   includeFields = config.entities.optimize ? config.entities.category.includeFields : null,
   excludeFields = config.entities.optimize ? config.entities.category.excludeFields : null
 }: DataResolver.CategorySearchOptions = {}): Promise<Category[]> => {
+  const allFilters = 'filterFields' in config.entities.category ? Object.assign(config.entities.category.filterFields, filters) : filters
```

Comment:
```suggestion
  const allFilters = config.entities.category.filterFields ? Object.assign({}, config.entities.category.filterFields, filters) : filters
```
we shouldn't try to assign into config property